css/styles.css

```diff
@@ -0,0 +1,86 @@
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body, p {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "helvetica neue", helvetica, arial, sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: .1em;
+  background: white;
+  text-transform: lowercase;
+}
+
+.homebox {
+  padding-top: 2em;
+  padding-left: 2em;
+  width: 50%;
+  /* border: 1px red solid; */
```

Comment:
we don't want this?

Reply:
we dont want the border, i was just using it to see what was up w my spacing and then turned it off, shall delete

Reply:
is this what you mean?

Reply:
yep!